view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -81,14 +81,22 @@ define(
 
                 let self = this;
 
-                let paymentMethodsObserver = adyenPaymentService.getPaymentMethods();
-                paymentMethodsObserver.subscribe(
-                    function (paymentMethodsResponse) {
-                        self.loadCheckoutComponent(paymentMethodsResponse)
-                    });
-
-                self.loadCheckoutComponent(paymentMethodsObserver());
-                return this;
+                adyenPaymentService.retrievePaymentMethods().done(function(paymentMethods) {
```

Comment:
This function may cause multiple `/paymentMethods` request on the same checkout. Is it possible to move this function to [multishipping/adyen-cc-method.js](https://github.com/Adyen/adyen-magento2/blob/rc-v9/view/frontend/web/js/view/payment/method-renderer/multishipping/adyen-cc-method.js)?